intro-HPC/ch_account.tex

```diff
@@ -300,15 +260,41 @@ \subsection{Users of the \association}
 Directory. (i.e., /Users/$<$username$>$/.ssh/id\_rsa.pub).
 
 
-\ifmac % IF MAC
+\fi % END IF ANTWERPEN
+
+\ifgent
+You will now have to log in with CAS using your UGent account, and then be requested
+to share your Mail and Affiliation with KU Leuven (which actually means the VSC accountpage)
+Select the 'Remember' checkbox and click 'Yes, continue'.
+\fi
+%TODO: other institute login information explained here?
```

Comment:
the other `\if<site>`'s should be here too, probably with a placeholder text/comment that makes it clear something is missing...